From 8f9c38735c87f406bbc4663e98e6563269f432fa Mon Sep 17 00:00:00 2001
From: Ariadne Conill <ariadne@dereferenced.org>
Date: Mon, 15 May 2023 23:20:16 +0000
Subject: [PATCH] Disable AVX512VL

The EVEX / AVX512VL optimized string instructions cause flaky behavior with
OpenJDK and others

Signed-off-by: Ariadne Conill <ariadne@dereferenced.org>
---
 sysdeps/x86/cpu-features.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sysdeps/x86/cpu-features.c b/sysdeps/x86/cpu-features.c
index 5bff8ec0b4..acfddb4def 100644
--- a/sysdeps/x86/cpu-features.c
+++ b/sysdeps/x86/cpu-features.c
@@ -176,8 +176,10 @@ update_active (struct cpu_features *cpu_features)
 		  CPU_FEATURE_SET_ACTIVE (cpu_features, AVX512ER);
 		  /* Determine if AVX512PF is usable.  */
 		  CPU_FEATURE_SET_ACTIVE (cpu_features, AVX512PF);
+#if 0
 		  /* Determine if AVX512VL is usable.  */
 		  CPU_FEATURE_SET_ACTIVE (cpu_features, AVX512VL);
+#endif
 		  /* Determine if AVX512DQ is usable.  */
 		  CPU_FEATURE_SET_ACTIVE (cpu_features, AVX512DQ);
 		  /* Determine if AVX512BW is usable.  */
-- 
2.40.0

